client/src/utils/is_empty_object.js

```diff
@@ -0,0 +1,3 @@
+export default function isEmptyObject(obj = {}) {
```

Comment:
This should default to an empty object. If you don't pass the function, it shouldn't return true